### ISSUE TITLE
Post capture tasks dialog shortcuts

### ISSUE THREAD

**Issue by Slind14** [ShareX/ShareX#1168]
I would love to further speed up the work with ShareX and see the biggest potential within the `Post capture tasks` dialog if used.
It would be great if the options could be with shortcuts. E.g. 

To open the image with the image editor:

`2` + `Enter`

![](https://i.imgur.com/lvGluo1.png)

Another thought could be presets that can be given a custom hotkey.

**Comment by Jaex** [comment]
How 11. upload check box gonna have hotkey then?

**Comment by Slind14** [comment]
alt + number or customizable.

**Comment by Jaex** [comment]
Customizable really? Are you have any idea how much difficult it would be.

**Comment by Slind14** [comment]
I can let you know what I would love to use as a user. How or if it all is up you. 

What about keys only for the "most" use ones or letters? (image editor `e`, watermark `m`, save `s`, save as `S`, print `p`..)

**Comment by Jaex** [comment]
By the way you know that you can make different hotkeys per that tasks in workflow right?

**Comment by Jaex** [comment]
Added "After upload" tab now:

![](https://dl.dropboxusercontent.com/u/14076298/ShareX/2015/12/ShareX.vshost_E4KKPQafO3.png)

**Comment by Slind14** [comment]
Yes, while I use the following currently I find my self needing the past capture task quite often. Ofc, I could create one for every action and print out a cheat sheet.
![](https://i.imgur.com/8xkecjq.png)

**Comment by Slind14** [comment]
wow, thanks :)

**Comment by Jaex** [comment]
Also your suggested hotkeys won't work because file name textbox have focus so when you press those buttons it will change file name.

**Comment by Slind14** [comment]
Would simple shortcuts for each of the selection boxes and tabs be a lot of work? The file name box could be unfocused with `esc` and `i` to focus it (insert mode).

**Comment by Jaex** [comment]
If you gonna press multiple buttons to unfocus etc. why you don't just mouse click them which will be faster and not require memorizing shortcuts.

**Comment by Slind14** [comment]
Ever played counter strike? Even with the presets (workflows) you can save, using the short cuts is not much slower and you are not forced into predefined sets.
The thing about short cuts is that you learn them passively. It is not like a workflow where you forgot the shortcut for. The dialog opens up and you see the shortcuts directly, as prefix, suffix, highlighted letter or whatever. Once you have done a few screenshots, you are faster than you ever can be with the mouse.
Now we can argue that you need the mouse anyways for the selection, but you could say the same for counter strike.
On another matter it allows you take the screenshot without needing to think about how you need it and what the shortcut was.

**Comment by QuAzI** [comment]
Traditionally Esc used for dialog cancellation in most applications.
In most games it is also used for game cancellation (through menu item or directly)
